content/en/continuous_integration/pipelines/custom.md

```diff
@@ -90,7 +91,17 @@ To send pipeline events programmatically to Datadog, ensure that your [`DD_API_K
    EOF
    {{< /code-block >}}
 
-1. After sending your pipeline event to Datadog, you can integrate additional event types such as `stage`, `job`, and `step`. For more information, see the [Send Pipeline Event endpoint][1].
+3. After sending your pipeline event to Datadog, you can integrate additional event types such as `stage`, `job`, and `step`. For more information, see the [Send Pipeline Event endpoint][1].
+
+## Running pipelines
+Pipeline events sent with the `status` set to `running` have the same `unique_id` as the final pipeline event.
+Additionally, running pipelines can be updated by adding more information while they are still running. Essentially, a pipeline consists of the following events:
+
+1. The initial running pipeline event with the `status` set to `running`.
+2. Optionally, `N` running pipelines with the same `unique_id` and the `status` set to `running` to update the pipeline with more information.
+3. The final pipeline event with a non `running` status and the same `unique_id`.
```

Comment:
```suggestion
3. The final pipeline event **without** a `running` status and the same `unique_id`.
```

Reply:
If possible, can we tell users what the status will be, instead of what it will **not** be?

Reply:
The end status may be `success`, `failed`, `canceled`, ... We do not know. We can either tell them it is not running or list every possible finished state